### PR TITLE
Ruby Code Snippet for Lambda SQS Integration

### PR DESCRIPTION
**Pull request by ssnskar**
*Issue #, if available:* https://github.com/aws-samples/serverless-snippets/issues/93

*Description of changes:* Ruby Code Snippet for Lambda SQS Integration

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integration-sqs-to-lambda/snippet-data.json [hunk 1]
```diff
@@ -102,7 +113,7 @@
       "linkedin": "girishbalachandran"
     },
     {
-      "headline": "Python Example Presented by Girish B",
+      "headline": "Python & Ruby Example Presented by Sanskar",
```
Comment (boyney123) [inline integration-sqs-to-lambda/snippet-data.json:116]: Did you provide both the python and ruby example @ssnskar ? What happened to Girish B?

**Comment by ssnskar** [comment]
Hey David, Yes I have provided the python code previously. And now the Ruby. 
Girish B provided the Java code. There was a typo mistake previously as I got the template from him.
I corrected it in this request.

**Comment by dpruessner** [comment]
@boyney123 Can you give guidance on the `$.snippets[0].codeTabs[].snippets[].language` value mapping for Ruby?  

Looks like many languages are using the file extension (eg, Python has `"language":"py"`), but .NET files with *.cs* extensions have `"language":"dotnet"`.  Is there a documented schema with an expected enum variant that we should be following?

I just approved a PR for Ruby with `"language":"ruby"`, since syntax-highlighting for Markdown code-blocks use 
      
      ```ruby

 as the label.

**Comment by boyney123** [comment]
Hey @dpruessner 

Yeah `ruby` should be fine, we use this list here https://prismjs.com/#supported-languages

**Comment by boyney123** [comment]
@ssnskar and @dpruessner are we good to merge this now? Had the ruby code been reviewed?

**Review by dpruessner — APPROVED** [review]

**Comment by dpruessner** [comment]
> @ssnskar and @dpruessner are we good to merge this now? Had the ruby code been reviewed?

Yep; good-to-go. Updating Asana

**Pull request merged**